README.md

```diff
@@ -2,40 +2,50 @@
 
 A terraform module to configure a [terraform remote state](https://www.terraform.io/docs/state/remote.html) for your AWS accounts.
 
+## Overview
+
+Terraform uses [state](https://www.terraform.io/docs/state/index.html) to keep track of your infrastructure and configuration. When working with Terraform in a team setting, it is recommend to use a [remote state](https://www.terraform.io/docs/state/remote.html). This module creates all of the necessary resources to keep your state files is a centralized remote location using an S3 [backend](https://www.terraform.io/docs/backends/index.html). It will also create DynamoDB table(s) that will track deployment locks thus preventing any accidental overlap.
+
 ## Usage
 
-Terraform uses [state](https://www.terraform.io/docs/state/index.html) in order to keep track of your infrastructure and configuration. When working with Terraform in a team setting, it is recommend to use [terraform remote state](https://www.terraform.io/docs/state/remote.html). The `terraform-aws-bootstrapper` module creates the necessary resources to store the `terraform.tfstate` file remotely using an S3 [backend](https://www.terraform.io/docs/backends/index.html) and DynamoDB. Therefore the `terraform.tfstate` file for the remote state configuration must be kept with the code repository in which it is deployed.
+This module serves as more of a blueprint to quicky create and configure all of the necessary resource to integrate your terraform project using an S3 backend.
```

Comment:
Typo: quicky create and configure all of the necessary resource (necessary resources)